2018-06-26 06:00:52 Error: [Cake\Controller\Exception\MissingActionException] Action TestsController::15() could not be found, or is not accessible.
Exception Attributes: array (
  'controller' => 'TestsController',
  'action' => '15',
  'prefix' => '',
  'plugin' => NULL,
)
Request URL: /tests/15/12
Stack Trace:
#0 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Http\ActionDispatcher.php(122): Cake\Controller\Controller->invokeAction()
#1 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Http\ActionDispatcher.php(96): Cake\Http\ActionDispatcher->_invoke(Object(App\Controller\TestsController))
#2 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Http\BaseApplication.php(83): Cake\Http\ActionDispatcher->dispatch(Object(Cake\Network\Request), Object(Cake\Network\Response))
#3 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Http\Runner.php(65): Cake\Http\BaseApplication->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response), Object(Cake\Http\Runner))
#4 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Routing\Middleware\RoutingMiddleware.php(62): Cake\Http\Runner->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#5 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Http\Runner.php(65): Cake\Routing\Middleware\RoutingMiddleware->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response), Object(Cake\Http\Runner))
#6 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Routing\Middleware\AssetMiddleware.php(88): Cake\Http\Runner->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#7 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Http\Runner.php(65): Cake\Routing\Middleware\AssetMiddleware->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response), Object(Cake\Http\Runner))
#8 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Error\Middleware\ErrorHandlerMiddleware.php(81): Cake\Http\Runner->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#9 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Http\Runner.php(65): Cake\Error\Middleware\ErrorHandlerMiddleware->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response), Object(Cake\Http\Runner))
#10 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Http\Runner.php(51): Cake\Http\Runner->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#11 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Http\Server.php(92): Cake\Http\Runner->run(Object(Cake\Http\MiddlewareQueue), Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#12 C:\xampp\htdocs\mycake\cakephp3\webroot\index.php(37): Cake\Http\Server->run()
#13 {main}


2018-06-26 06:10:55 Error: [Cake\Routing\Exception\MissingControllerException] Controller class Generate could not be found.
Exception Attributes: array (
  'class' => 'Generate',
  'plugin' => false,
  'prefix' => false,
  '_ext' => false,
)
Request URL: /generate
Stack Trace:
#0 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Http\ControllerFactory.php(72): Cake\Http\ControllerFactory->missingController(Object(Cake\Network\Request))
#1 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Http\ActionDispatcher.php(93): Cake\Http\ControllerFactory->create(Object(Cake\Network\Request), Object(Cake\Network\Response))
#2 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Http\BaseApplication.php(83): Cake\Http\ActionDispatcher->dispatch(Object(Cake\Network\Request), Object(Cake\Network\Response))
#3 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Http\Runner.php(65): Cake\Http\BaseApplication->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response), Object(Cake\Http\Runner))
#4 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Routing\Middleware\RoutingMiddleware.php(62): Cake\Http\Runner->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#5 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Http\Runner.php(65): Cake\Routing\Middleware\RoutingMiddleware->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response), Object(Cake\Http\Runner))
#6 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Routing\Middleware\AssetMiddleware.php(88): Cake\Http\Runner->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#7 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Http\Runner.php(65): Cake\Routing\Middleware\AssetMiddleware->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response), Object(Cake\Http\Runner))
#8 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Error\Middleware\ErrorHandlerMiddleware.php(81): Cake\Http\Runner->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#9 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Http\Runner.php(65): Cake\Error\Middleware\ErrorHandlerMiddleware->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response), Object(Cake\Http\Runner))
#10 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Http\Runner.php(51): Cake\Http\Runner->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#11 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Http\Server.php(92): Cake\Http\Runner->run(Object(Cake\Http\MiddlewareQueue), Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#12 C:\xampp\htdocs\mycake\cakephp3\webroot\index.php(37): Cake\Http\Server->run()
#13 {main}


2018-06-26 06:18:53 Error: [Cake\View\Exception\MissingTemplateException] Template file "Generates\maxmil.ctp" is missing.
Exception Attributes: array (
  'file' => 'Generates\\maxmil.ctp',
)
Request URL: /gen
Stack Trace:
#0 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\View\View.php(593): Cake\View\View->_getViewFileName(NULL)
#1 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Controller\Controller.php(617): Cake\View\View->render(NULL, NULL)
#2 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Http\ActionDispatcher.php(128): Cake\Controller\Controller->render()
#3 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Http\ActionDispatcher.php(96): Cake\Http\ActionDispatcher->_invoke(Object(App\Controller\GeneratesController))
#4 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Http\BaseApplication.php(83): Cake\Http\ActionDispatcher->dispatch(Object(Cake\Network\Request), Object(Cake\Network\Response))
#5 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Http\Runner.php(65): Cake\Http\BaseApplication->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response), Object(Cake\Http\Runner))
#6 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Routing\Middleware\RoutingMiddleware.php(62): Cake\Http\Runner->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#7 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Http\Runner.php(65): Cake\Routing\Middleware\RoutingMiddleware->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response), Object(Cake\Http\Runner))
#8 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Routing\Middleware\AssetMiddleware.php(88): Cake\Http\Runner->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#9 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Http\Runner.php(65): Cake\Routing\Middleware\AssetMiddleware->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response), Object(Cake\Http\Runner))
#10 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Error\Middleware\ErrorHandlerMiddleware.php(81): Cake\Http\Runner->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#11 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Http\Runner.php(65): Cake\Error\Middleware\ErrorHandlerMiddleware->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response), Object(Cake\Http\Runner))
#12 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Http\Runner.php(51): Cake\Http\Runner->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#13 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Http\Server.php(92): Cake\Http\Runner->run(Object(Cake\Http\MiddlewareQueue), Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#14 C:\xampp\htdocs\mycake\cakephp3\webroot\index.php(37): Cake\Http\Server->run()
#15 {main}


2018-06-26 07:06:44 Error: [Cake\Routing\Exception\MissingControllerException] Controller class RedirectController could not be found.
Exception Attributes: array (
  'class' => 'RedirectController',
  'plugin' => false,
  'prefix' => false,
  '_ext' => false,
)
Request URL: /redirect-controller
Stack Trace:
#0 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Http\ControllerFactory.php(72): Cake\Http\ControllerFactory->missingController(Object(Cake\Network\Request))
#1 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Http\ActionDispatcher.php(93): Cake\Http\ControllerFactory->create(Object(Cake\Network\Request), Object(Cake\Network\Response))
#2 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Http\BaseApplication.php(83): Cake\Http\ActionDispatcher->dispatch(Object(Cake\Network\Request), Object(Cake\Network\Response))
#3 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Http\Runner.php(65): Cake\Http\BaseApplication->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response), Object(Cake\Http\Runner))
#4 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Routing\Middleware\RoutingMiddleware.php(62): Cake\Http\Runner->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#5 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Http\Runner.php(65): Cake\Routing\Middleware\RoutingMiddleware->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response), Object(Cake\Http\Runner))
#6 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Routing\Middleware\AssetMiddleware.php(88): Cake\Http\Runner->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#7 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Http\Runner.php(65): Cake\Routing\Middleware\AssetMiddleware->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response), Object(Cake\Http\Runner))
#8 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Error\Middleware\ErrorHandlerMiddleware.php(81): Cake\Http\Runner->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#9 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Http\Runner.php(65): Cake\Error\Middleware\ErrorHandlerMiddleware->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response), Object(Cake\Http\Runner))
#10 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Http\Runner.php(51): Cake\Http\Runner->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#11 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Http\Server.php(92): Cake\Http\Runner->run(Object(Cake\Http\MiddlewareQueue), Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#12 C:\xampp\htdocs\mycake\cakephp3\webroot\index.php(37): Cake\Http\Server->run()
#13 {main}


2018-06-26 07:08:49 Error: [Cake\Routing\Exception\MissingControllerException] Controller class RedirectController could not be found.
Exception Attributes: array (
  'class' => 'RedirectController',
  'plugin' => false,
  'prefix' => false,
  '_ext' => false,
)
Request URL: /redirect-controller
Stack Trace:
#0 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Http\ControllerFactory.php(72): Cake\Http\ControllerFactory->missingController(Object(Cake\Network\Request))
#1 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Http\ActionDispatcher.php(93): Cake\Http\ControllerFactory->create(Object(Cake\Network\Request), Object(Cake\Network\Response))
#2 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Http\BaseApplication.php(83): Cake\Http\ActionDispatcher->dispatch(Object(Cake\Network\Request), Object(Cake\Network\Response))
#3 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Http\Runner.php(65): Cake\Http\BaseApplication->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response), Object(Cake\Http\Runner))
#4 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Routing\Middleware\RoutingMiddleware.php(62): Cake\Http\Runner->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#5 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Http\Runner.php(65): Cake\Routing\Middleware\RoutingMiddleware->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response), Object(Cake\Http\Runner))
#6 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Routing\Middleware\AssetMiddleware.php(88): Cake\Http\Runner->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#7 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Http\Runner.php(65): Cake\Routing\Middleware\AssetMiddleware->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response), Object(Cake\Http\Runner))
#8 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Error\Middleware\ErrorHandlerMiddleware.php(81): Cake\Http\Runner->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#9 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Http\Runner.php(65): Cake\Error\Middleware\ErrorHandlerMiddleware->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response), Object(Cake\Http\Runner))
#10 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Http\Runner.php(51): Cake\Http\Runner->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#11 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Http\Server.php(92): Cake\Http\Runner->run(Object(Cake\Http\MiddlewareQueue), Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#12 C:\xampp\htdocs\mycake\cakephp3\webroot\index.php(37): Cake\Http\Server->run()
#13 {main}


2018-06-26 07:13:06 Error: [Cake\Controller\Exception\MissingActionException] Action Controller::action1() could not be found, or is not accessible.
Exception Attributes: array (
  'controller' => 'Controller',
  'action' => 'action1',
  'prefix' => '',
  'plugin' => NULL,
)
Request URL: /redirectcontroller
Stack Trace:
#0 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Http\ActionDispatcher.php(122): Cake\Controller\Controller->invokeAction()
#1 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Http\ActionDispatcher.php(96): Cake\Http\ActionDispatcher->_invoke(Object(Cake\Controller\Controller))
#2 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Http\BaseApplication.php(83): Cake\Http\ActionDispatcher->dispatch(Object(Cake\Network\Request), Object(Cake\Network\Response))
#3 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Http\Runner.php(65): Cake\Http\BaseApplication->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response), Object(Cake\Http\Runner))
#4 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Routing\Middleware\RoutingMiddleware.php(62): Cake\Http\Runner->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#5 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Http\Runner.php(65): Cake\Routing\Middleware\RoutingMiddleware->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response), Object(Cake\Http\Runner))
#6 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Routing\Middleware\AssetMiddleware.php(88): Cake\Http\Runner->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#7 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Http\Runner.php(65): Cake\Routing\Middleware\AssetMiddleware->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response), Object(Cake\Http\Runner))
#8 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Error\Middleware\ErrorHandlerMiddleware.php(81): Cake\Http\Runner->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#9 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Http\Runner.php(65): Cake\Error\Middleware\ErrorHandlerMiddleware->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response), Object(Cake\Http\Runner))
#10 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Http\Runner.php(51): Cake\Http\Runner->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#11 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Http\Server.php(92): Cake\Http\Runner->run(Object(Cake\Http\MiddlewareQueue), Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#12 C:\xampp\htdocs\mycake\cakephp3\webroot\index.php(37): Cake\Http\Server->run()
#13 {main}


2018-06-26 07:13:46 Error: [Cake\Routing\Exception\MissingControllerException] Controller class Redirectcontroller2 could not be found.
Exception Attributes: array (
  'class' => 'Redirectcontroller2',
  'plugin' => false,
  'prefix' => false,
  '_ext' => false,
)
Request URL: /redirectcontroller-2
Stack Trace:
#0 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Http\ControllerFactory.php(72): Cake\Http\ControllerFactory->missingController(Object(Cake\Network\Request))
#1 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Http\ActionDispatcher.php(93): Cake\Http\ControllerFactory->create(Object(Cake\Network\Request), Object(Cake\Network\Response))
#2 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Http\BaseApplication.php(83): Cake\Http\ActionDispatcher->dispatch(Object(Cake\Network\Request), Object(Cake\Network\Response))
#3 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Http\Runner.php(65): Cake\Http\BaseApplication->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response), Object(Cake\Http\Runner))
#4 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Routing\Middleware\RoutingMiddleware.php(62): Cake\Http\Runner->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#5 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Http\Runner.php(65): Cake\Routing\Middleware\RoutingMiddleware->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response), Object(Cake\Http\Runner))
#6 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Routing\Middleware\AssetMiddleware.php(88): Cake\Http\Runner->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#7 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Http\Runner.php(65): Cake\Routing\Middleware\AssetMiddleware->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response), Object(Cake\Http\Runner))
#8 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Error\Middleware\ErrorHandlerMiddleware.php(81): Cake\Http\Runner->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#9 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Http\Runner.php(65): Cake\Error\Middleware\ErrorHandlerMiddleware->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response), Object(Cake\Http\Runner))
#10 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Http\Runner.php(51): Cake\Http\Runner->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#11 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Http\Server.php(92): Cake\Http\Runner->run(Object(Cake\Http\MiddlewareQueue), Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#12 C:\xampp\htdocs\mycake\cakephp3\webroot\index.php(37): Cake\Http\Server->run()
#13 {main}


2018-06-26 07:14:00 Error: [Cake\Controller\Exception\MissingActionException] Action Controller::action2() could not be found, or is not accessible.
Exception Attributes: array (
  'controller' => 'Controller',
  'action' => 'action2',
  'prefix' => '',
  'plugin' => NULL,
)
Request URL: /redirectcontroller2
Stack Trace:
#0 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Http\ActionDispatcher.php(122): Cake\Controller\Controller->invokeAction()
#1 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Http\ActionDispatcher.php(96): Cake\Http\ActionDispatcher->_invoke(Object(Cake\Controller\Controller))
#2 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Http\BaseApplication.php(83): Cake\Http\ActionDispatcher->dispatch(Object(Cake\Network\Request), Object(Cake\Network\Response))
#3 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Http\Runner.php(65): Cake\Http\BaseApplication->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response), Object(Cake\Http\Runner))
#4 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Routing\Middleware\RoutingMiddleware.php(62): Cake\Http\Runner->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#5 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Http\Runner.php(65): Cake\Routing\Middleware\RoutingMiddleware->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response), Object(Cake\Http\Runner))
#6 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Routing\Middleware\AssetMiddleware.php(88): Cake\Http\Runner->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#7 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Http\Runner.php(65): Cake\Routing\Middleware\AssetMiddleware->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response), Object(Cake\Http\Runner))
#8 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Error\Middleware\ErrorHandlerMiddleware.php(81): Cake\Http\Runner->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#9 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Http\Runner.php(65): Cake\Error\Middleware\ErrorHandlerMiddleware->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response), Object(Cake\Http\Runner))
#10 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Http\Runner.php(51): Cake\Http\Runner->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#11 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Http\Server.php(92): Cake\Http\Runner->run(Object(Cake\Http\MiddlewareQueue), Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#12 C:\xampp\htdocs\mycake\cakephp3\webroot\index.php(37): Cake\Http\Server->run()
#13 {main}


2018-06-26 07:23:44 Error: [Cake\Controller\Exception\MissingActionException] Action Controller::action2() could not be found, or is not accessible.
Exception Attributes: array (
  'controller' => 'Controller',
  'action' => 'action2',
  'prefix' => '',
  'plugin' => NULL,
)
Request URL: /redirectcontroller2
Stack Trace:
#0 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Http\ActionDispatcher.php(122): Cake\Controller\Controller->invokeAction()
#1 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Http\ActionDispatcher.php(96): Cake\Http\ActionDispatcher->_invoke(Object(Cake\Controller\Controller))
#2 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Http\BaseApplication.php(83): Cake\Http\ActionDispatcher->dispatch(Object(Cake\Network\Request), Object(Cake\Network\Response))
#3 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Http\Runner.php(65): Cake\Http\BaseApplication->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response), Object(Cake\Http\Runner))
#4 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Routing\Middleware\RoutingMiddleware.php(62): Cake\Http\Runner->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#5 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Http\Runner.php(65): Cake\Routing\Middleware\RoutingMiddleware->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response), Object(Cake\Http\Runner))
#6 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Routing\Middleware\AssetMiddleware.php(88): Cake\Http\Runner->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#7 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Http\Runner.php(65): Cake\Routing\Middleware\AssetMiddleware->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response), Object(Cake\Http\Runner))
#8 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Error\Middleware\ErrorHandlerMiddleware.php(81): Cake\Http\Runner->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#9 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Http\Runner.php(65): Cake\Error\Middleware\ErrorHandlerMiddleware->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response), Object(Cake\Http\Runner))
#10 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Http\Runner.php(51): Cake\Http\Runner->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#11 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Http\Server.php(92): Cake\Http\Runner->run(Object(Cake\Http\MiddlewareQueue), Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#12 C:\xampp\htdocs\mycake\cakephp3\webroot\index.php(37): Cake\Http\Server->run()
#13 {main}


2018-06-26 07:26:42 Error: [Cake\Controller\Exception\MissingActionException] Action Controller::action2() could not be found, or is not accessible.
Exception Attributes: array (
  'controller' => 'Controller',
  'action' => 'action2',
  'prefix' => '',
  'plugin' => NULL,
)
Request URL: /redirectcontroller2
Stack Trace:
#0 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Http\ActionDispatcher.php(122): Cake\Controller\Controller->invokeAction()
#1 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Http\ActionDispatcher.php(96): Cake\Http\ActionDispatcher->_invoke(Object(Cake\Controller\Controller))
#2 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Http\BaseApplication.php(83): Cake\Http\ActionDispatcher->dispatch(Object(Cake\Network\Request), Object(Cake\Network\Response))
#3 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Http\Runner.php(65): Cake\Http\BaseApplication->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response), Object(Cake\Http\Runner))
#4 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Routing\Middleware\RoutingMiddleware.php(62): Cake\Http\Runner->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#5 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Http\Runner.php(65): Cake\Routing\Middleware\RoutingMiddleware->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response), Object(Cake\Http\Runner))
#6 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Routing\Middleware\AssetMiddleware.php(88): Cake\Http\Runner->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#7 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Http\Runner.php(65): Cake\Routing\Middleware\AssetMiddleware->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response), Object(Cake\Http\Runner))
#8 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Error\Middleware\ErrorHandlerMiddleware.php(81): Cake\Http\Runner->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#9 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Http\Runner.php(65): Cake\Error\Middleware\ErrorHandlerMiddleware->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response), Object(Cake\Http\Runner))
#10 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Http\Runner.php(51): Cake\Http\Runner->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#11 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Http\Server.php(92): Cake\Http\Runner->run(Object(Cake\Http\MiddlewareQueue), Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#12 C:\xampp\htdocs\mycake\cakephp3\webroot\index.php(37): Cake\Http\Server->run()
#13 {main}


2018-06-26 07:26:50 Error: [Cake\Routing\Exception\MissingControllerException] Controller class RedirectController could not be found.
Exception Attributes: array (
  'class' => 'RedirectController',
  'plugin' => false,
  'prefix' => false,
  '_ext' => false,
)
Request URL: /redirect-controller
Stack Trace:
#0 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Http\ControllerFactory.php(72): Cake\Http\ControllerFactory->missingController(Object(Cake\Network\Request))
#1 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Http\ActionDispatcher.php(93): Cake\Http\ControllerFactory->create(Object(Cake\Network\Request), Object(Cake\Network\Response))
#2 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Http\BaseApplication.php(83): Cake\Http\ActionDispatcher->dispatch(Object(Cake\Network\Request), Object(Cake\Network\Response))
#3 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Http\Runner.php(65): Cake\Http\BaseApplication->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response), Object(Cake\Http\Runner))
#4 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Routing\Middleware\RoutingMiddleware.php(62): Cake\Http\Runner->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#5 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Http\Runner.php(65): Cake\Routing\Middleware\RoutingMiddleware->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response), Object(Cake\Http\Runner))
#6 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Routing\Middleware\AssetMiddleware.php(88): Cake\Http\Runner->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#7 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Http\Runner.php(65): Cake\Routing\Middleware\AssetMiddleware->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response), Object(Cake\Http\Runner))
#8 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Error\Middleware\ErrorHandlerMiddleware.php(81): Cake\Http\Runner->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#9 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Http\Runner.php(65): Cake\Error\Middleware\ErrorHandlerMiddleware->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response), Object(Cake\Http\Runner))
#10 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Http\Runner.php(51): Cake\Http\Runner->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#11 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Http\Server.php(92): Cake\Http\Runner->run(Object(Cake\Http\MiddlewareQueue), Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#12 C:\xampp\htdocs\mycake\cakephp3\webroot\index.php(37): Cake\Http\Server->run()
#13 {main}


2018-06-26 07:26:59 Error: [Cake\Controller\Exception\MissingActionException] Action Controller::action1() could not be found, or is not accessible.
Exception Attributes: array (
  'controller' => 'Controller',
  'action' => 'action1',
  'prefix' => '',
  'plugin' => NULL,
)
Request URL: /redirectcontroller
Stack Trace:
#0 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Http\ActionDispatcher.php(122): Cake\Controller\Controller->invokeAction()
#1 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Http\ActionDispatcher.php(96): Cake\Http\ActionDispatcher->_invoke(Object(Cake\Controller\Controller))
#2 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Http\BaseApplication.php(83): Cake\Http\ActionDispatcher->dispatch(Object(Cake\Network\Request), Object(Cake\Network\Response))
#3 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Http\Runner.php(65): Cake\Http\BaseApplication->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response), Object(Cake\Http\Runner))
#4 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Routing\Middleware\RoutingMiddleware.php(62): Cake\Http\Runner->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#5 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Http\Runner.php(65): Cake\Routing\Middleware\RoutingMiddleware->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response), Object(Cake\Http\Runner))
#6 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Routing\Middleware\AssetMiddleware.php(88): Cake\Http\Runner->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#7 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Http\Runner.php(65): Cake\Routing\Middleware\AssetMiddleware->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response), Object(Cake\Http\Runner))
#8 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Error\Middleware\ErrorHandlerMiddleware.php(81): Cake\Http\Runner->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#9 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Http\Runner.php(65): Cake\Error\Middleware\ErrorHandlerMiddleware->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response), Object(Cake\Http\Runner))
#10 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Http\Runner.php(51): Cake\Http\Runner->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#11 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Http\Server.php(92): Cake\Http\Runner->run(Object(Cake\Http\MiddlewareQueue), Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#12 C:\xampp\htdocs\mycake\cakephp3\webroot\index.php(37): Cake\Http\Server->run()
#13 {main}


2018-06-26 07:32:52 Error: [Cake\Controller\Exception\MissingActionException] Action Controller::action1() could not be found, or is not accessible.
Exception Attributes: array (
  'controller' => 'Controller',
  'action' => 'action1',
  'prefix' => '',
  'plugin' => NULL,
)
Request URL: /redirectcontroller
Stack Trace:
#0 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Http\ActionDispatcher.php(122): Cake\Controller\Controller->invokeAction()
#1 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Http\ActionDispatcher.php(96): Cake\Http\ActionDispatcher->_invoke(Object(Cake\Controller\Controller))
#2 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Http\BaseApplication.php(83): Cake\Http\ActionDispatcher->dispatch(Object(Cake\Network\Request), Object(Cake\Network\Response))
#3 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Http\Runner.php(65): Cake\Http\BaseApplication->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response), Object(Cake\Http\Runner))
#4 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Routing\Middleware\RoutingMiddleware.php(62): Cake\Http\Runner->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#5 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Http\Runner.php(65): Cake\Routing\Middleware\RoutingMiddleware->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response), Object(Cake\Http\Runner))
#6 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Routing\Middleware\AssetMiddleware.php(88): Cake\Http\Runner->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#7 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Http\Runner.php(65): Cake\Routing\Middleware\AssetMiddleware->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response), Object(Cake\Http\Runner))
#8 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Error\Middleware\ErrorHandlerMiddleware.php(81): Cake\Http\Runner->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#9 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Http\Runner.php(65): Cake\Error\Middleware\ErrorHandlerMiddleware->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response), Object(Cake\Http\Runner))
#10 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Http\Runner.php(51): Cake\Http\Runner->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#11 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Http\Server.php(92): Cake\Http\Runner->run(Object(Cake\Http\MiddlewareQueue), Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#12 C:\xampp\htdocs\mycake\cakephp3\webroot\index.php(37): Cake\Http\Server->run()
#13 {main}


2018-06-26 07:33:02 Error: [Cake\Controller\Exception\MissingActionException] Action Controller::action2() could not be found, or is not accessible.
Exception Attributes: array (
  'controller' => 'Controller',
  'action' => 'action2',
  'prefix' => '',
  'plugin' => NULL,
)
Request URL: /redirectcontroller2
Stack Trace:
#0 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Http\ActionDispatcher.php(122): Cake\Controller\Controller->invokeAction()
#1 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Http\ActionDispatcher.php(96): Cake\Http\ActionDispatcher->_invoke(Object(Cake\Controller\Controller))
#2 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Http\BaseApplication.php(83): Cake\Http\ActionDispatcher->dispatch(Object(Cake\Network\Request), Object(Cake\Network\Response))
#3 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Http\Runner.php(65): Cake\Http\BaseApplication->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response), Object(Cake\Http\Runner))
#4 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Routing\Middleware\RoutingMiddleware.php(62): Cake\Http\Runner->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#5 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Http\Runner.php(65): Cake\Routing\Middleware\RoutingMiddleware->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response), Object(Cake\Http\Runner))
#6 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Routing\Middleware\AssetMiddleware.php(88): Cake\Http\Runner->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#7 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Http\Runner.php(65): Cake\Routing\Middleware\AssetMiddleware->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response), Object(Cake\Http\Runner))
#8 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Error\Middleware\ErrorHandlerMiddleware.php(81): Cake\Http\Runner->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#9 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Http\Runner.php(65): Cake\Error\Middleware\ErrorHandlerMiddleware->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response), Object(Cake\Http\Runner))
#10 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Http\Runner.php(51): Cake\Http\Runner->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#11 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Http\Server.php(92): Cake\Http\Runner->run(Object(Cake\Http\MiddlewareQueue), Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#12 C:\xampp\htdocs\mycake\cakephp3\webroot\index.php(37): Cake\Http\Server->run()
#13 {main}


2018-06-26 07:34:59 Error: [Cake\Controller\Exception\MissingActionException] Action Controller::action2() could not be found, or is not accessible.
Exception Attributes: array (
  'controller' => 'Controller',
  'action' => 'action2',
  'prefix' => '',
  'plugin' => NULL,
)
Request URL: /redirectcontroller2
Stack Trace:
#0 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Http\ActionDispatcher.php(122): Cake\Controller\Controller->invokeAction()
#1 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Http\ActionDispatcher.php(96): Cake\Http\ActionDispatcher->_invoke(Object(Cake\Controller\Controller))
#2 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Http\BaseApplication.php(83): Cake\Http\ActionDispatcher->dispatch(Object(Cake\Network\Request), Object(Cake\Network\Response))
#3 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Http\Runner.php(65): Cake\Http\BaseApplication->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response), Object(Cake\Http\Runner))
#4 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Routing\Middleware\RoutingMiddleware.php(62): Cake\Http\Runner->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#5 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Http\Runner.php(65): Cake\Routing\Middleware\RoutingMiddleware->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response), Object(Cake\Http\Runner))
#6 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Routing\Middleware\AssetMiddleware.php(88): Cake\Http\Runner->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#7 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Http\Runner.php(65): Cake\Routing\Middleware\AssetMiddleware->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response), Object(Cake\Http\Runner))
#8 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Error\Middleware\ErrorHandlerMiddleware.php(81): Cake\Http\Runner->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#9 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Http\Runner.php(65): Cake\Error\Middleware\ErrorHandlerMiddleware->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response), Object(Cake\Http\Runner))
#10 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Http\Runner.php(51): Cake\Http\Runner->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#11 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Http\Server.php(92): Cake\Http\Runner->run(Object(Cake\Http\MiddlewareQueue), Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#12 C:\xampp\htdocs\mycake\cakephp3\webroot\index.php(37): Cake\Http\Server->run()
#13 {main}


2018-06-26 07:35:33 Error: [Cake\Controller\Exception\MissingActionException] Action Controller::action2() could not be found, or is not accessible.
Exception Attributes: array (
  'controller' => 'Controller',
  'action' => 'action2',
  'prefix' => '',
  'plugin' => NULL,
)
Request URL: /redirectcontroller2
Stack Trace:
#0 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Http\ActionDispatcher.php(122): Cake\Controller\Controller->invokeAction()
#1 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Http\ActionDispatcher.php(96): Cake\Http\ActionDispatcher->_invoke(Object(Cake\Controller\Controller))
#2 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Http\BaseApplication.php(83): Cake\Http\ActionDispatcher->dispatch(Object(Cake\Network\Request), Object(Cake\Network\Response))
#3 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Http\Runner.php(65): Cake\Http\BaseApplication->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response), Object(Cake\Http\Runner))
#4 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Routing\Middleware\RoutingMiddleware.php(62): Cake\Http\Runner->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#5 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Http\Runner.php(65): Cake\Routing\Middleware\RoutingMiddleware->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response), Object(Cake\Http\Runner))
#6 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Routing\Middleware\AssetMiddleware.php(88): Cake\Http\Runner->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#7 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Http\Runner.php(65): Cake\Routing\Middleware\AssetMiddleware->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response), Object(Cake\Http\Runner))
#8 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Error\Middleware\ErrorHandlerMiddleware.php(81): Cake\Http\Runner->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#9 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Http\Runner.php(65): Cake\Error\Middleware\ErrorHandlerMiddleware->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response), Object(Cake\Http\Runner))
#10 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Http\Runner.php(51): Cake\Http\Runner->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#11 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Http\Server.php(92): Cake\Http\Runner->run(Object(Cake\Http\MiddlewareQueue), Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#12 C:\xampp\htdocs\mycake\cakephp3\webroot\index.php(37): Cake\Http\Server->run()
#13 {main}


2018-06-26 07:35:59 Error: [Cake\Routing\Exception\MissingRouteException] A route matching "/" could not be found.
Exception Attributes: array (
  'url' => '/',
)
Request URL: /
Stack Trace:
#0 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Routing\Router.php(351): Cake\Routing\RouteCollection->parse('/', 'GET')
#1 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Routing\Middleware\RoutingMiddleware.php(51): Cake\Routing\Router::parse('/', 'GET')
#2 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Http\Runner.php(65): Cake\Routing\Middleware\RoutingMiddleware->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response), Object(Cake\Http\Runner))
#3 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Routing\Middleware\AssetMiddleware.php(88): Cake\Http\Runner->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#4 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Http\Runner.php(65): Cake\Routing\Middleware\AssetMiddleware->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response), Object(Cake\Http\Runner))
#5 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Error\Middleware\ErrorHandlerMiddleware.php(81): Cake\Http\Runner->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#6 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Http\Runner.php(65): Cake\Error\Middleware\ErrorHandlerMiddleware->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response), Object(Cake\Http\Runner))
#7 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Http\Runner.php(51): Cake\Http\Runner->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#8 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Http\Server.php(92): Cake\Http\Runner->run(Object(Cake\Http\MiddlewareQueue), Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#9 C:\xampp\htdocs\mycake\cakephp3\webroot\index.php(37): Cake\Http\Server->run()
#10 {main}


2018-06-26 07:37:01 Error: [Cake\Routing\Exception\MissingRouteException] A route matching "/" could not be found.
Exception Attributes: array (
  'url' => '/',
)
Request URL: /
Stack Trace:
#0 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Routing\Router.php(351): Cake\Routing\RouteCollection->parse('/', 'GET')
#1 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Routing\Middleware\RoutingMiddleware.php(51): Cake\Routing\Router::parse('/', 'GET')
#2 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Http\Runner.php(65): Cake\Routing\Middleware\RoutingMiddleware->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response), Object(Cake\Http\Runner))
#3 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Routing\Middleware\AssetMiddleware.php(88): Cake\Http\Runner->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#4 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Http\Runner.php(65): Cake\Routing\Middleware\AssetMiddleware->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response), Object(Cake\Http\Runner))
#5 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Error\Middleware\ErrorHandlerMiddleware.php(81): Cake\Http\Runner->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#6 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Http\Runner.php(65): Cake\Error\Middleware\ErrorHandlerMiddleware->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response), Object(Cake\Http\Runner))
#7 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Http\Runner.php(51): Cake\Http\Runner->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#8 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Http\Server.php(92): Cake\Http\Runner->run(Object(Cake\Http\MiddlewareQueue), Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#9 C:\xampp\htdocs\mycake\cakephp3\webroot\index.php(37): Cake\Http\Server->run()
#10 {main}


2018-06-26 08:48:50 Error: [Cake\Controller\Exception\MissingActionException] Action Controller::action2() could not be found, or is not accessible.
Exception Attributes: array (
  'controller' => 'Controller',
  'action' => 'action2',
  'prefix' => '',
  'plugin' => NULL,
)
Request URL: /redirectcontroller2
Stack Trace:
#0 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Http\ActionDispatcher.php(122): Cake\Controller\Controller->invokeAction()
#1 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Http\ActionDispatcher.php(96): Cake\Http\ActionDispatcher->_invoke(Object(Cake\Controller\Controller))
#2 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Http\BaseApplication.php(83): Cake\Http\ActionDispatcher->dispatch(Object(Cake\Network\Request), Object(Cake\Network\Response))
#3 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Http\Runner.php(65): Cake\Http\BaseApplication->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response), Object(Cake\Http\Runner))
#4 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Routing\Middleware\RoutingMiddleware.php(62): Cake\Http\Runner->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#5 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Http\Runner.php(65): Cake\Routing\Middleware\RoutingMiddleware->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response), Object(Cake\Http\Runner))
#6 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Routing\Middleware\AssetMiddleware.php(88): Cake\Http\Runner->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#7 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Http\Runner.php(65): Cake\Routing\Middleware\AssetMiddleware->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response), Object(Cake\Http\Runner))
#8 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Error\Middleware\ErrorHandlerMiddleware.php(81): Cake\Http\Runner->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#9 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Http\Runner.php(65): Cake\Error\Middleware\ErrorHandlerMiddleware->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response), Object(Cake\Http\Runner))
#10 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Http\Runner.php(51): Cake\Http\Runner->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#11 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Http\Server.php(92): Cake\Http\Runner->run(Object(Cake\Http\MiddlewareQueue), Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#12 C:\xampp\htdocs\mycake\cakephp3\webroot\index.php(37): Cake\Http\Server->run()
#13 {main}


2018-06-26 08:51:23 Error: [Cake\Controller\Exception\MissingActionException] Action Controller::action2() could not be found, or is not accessible.
Exception Attributes: array (
  'controller' => 'Controller',
  'action' => 'action2',
  'prefix' => '',
  'plugin' => NULL,
)
Request URL: /redirectcontroller2
Stack Trace:
#0 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Http\ActionDispatcher.php(122): Cake\Controller\Controller->invokeAction()
#1 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Http\ActionDispatcher.php(96): Cake\Http\ActionDispatcher->_invoke(Object(Cake\Controller\Controller))
#2 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Http\BaseApplication.php(83): Cake\Http\ActionDispatcher->dispatch(Object(Cake\Network\Request), Object(Cake\Network\Response))
#3 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Http\Runner.php(65): Cake\Http\BaseApplication->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response), Object(Cake\Http\Runner))
#4 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Routing\Middleware\RoutingMiddleware.php(62): Cake\Http\Runner->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#5 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Http\Runner.php(65): Cake\Routing\Middleware\RoutingMiddleware->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response), Object(Cake\Http\Runner))
#6 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Routing\Middleware\AssetMiddleware.php(88): Cake\Http\Runner->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#7 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Http\Runner.php(65): Cake\Routing\Middleware\AssetMiddleware->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response), Object(Cake\Http\Runner))
#8 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Error\Middleware\ErrorHandlerMiddleware.php(81): Cake\Http\Runner->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#9 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Http\Runner.php(65): Cake\Error\Middleware\ErrorHandlerMiddleware->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response), Object(Cake\Http\Runner))
#10 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Http\Runner.php(51): Cake\Http\Runner->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#11 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Http\Server.php(92): Cake\Http\Runner->run(Object(Cake\Http\MiddlewareQueue), Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#12 C:\xampp\htdocs\mycake\cakephp3\webroot\index.php(37): Cake\Http\Server->run()
#13 {main}


2018-06-26 08:56:56 Error: [Cake\Controller\Exception\MissingActionException] Action Controller::action2() could not be found, or is not accessible.
Exception Attributes: array (
  'controller' => 'Controller',
  'action' => 'action2',
  'prefix' => '',
  'plugin' => NULL,
)
Request URL: /redirectcontroller2
Stack Trace:
#0 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Http\ActionDispatcher.php(122): Cake\Controller\Controller->invokeAction()
#1 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Http\ActionDispatcher.php(96): Cake\Http\ActionDispatcher->_invoke(Object(Cake\Controller\Controller))
#2 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Http\BaseApplication.php(83): Cake\Http\ActionDispatcher->dispatch(Object(Cake\Network\Request), Object(Cake\Network\Response))
#3 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Http\Runner.php(65): Cake\Http\BaseApplication->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response), Object(Cake\Http\Runner))
#4 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Routing\Middleware\RoutingMiddleware.php(62): Cake\Http\Runner->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#5 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Http\Runner.php(65): Cake\Routing\Middleware\RoutingMiddleware->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response), Object(Cake\Http\Runner))
#6 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Routing\Middleware\AssetMiddleware.php(88): Cake\Http\Runner->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#7 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Http\Runner.php(65): Cake\Routing\Middleware\AssetMiddleware->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response), Object(Cake\Http\Runner))
#8 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Error\Middleware\ErrorHandlerMiddleware.php(81): Cake\Http\Runner->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#9 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Http\Runner.php(65): Cake\Error\Middleware\ErrorHandlerMiddleware->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response), Object(Cake\Http\Runner))
#10 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Http\Runner.php(51): Cake\Http\Runner->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#11 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Http\Server.php(92): Cake\Http\Runner->run(Object(Cake\Http\MiddlewareQueue), Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#12 C:\xampp\htdocs\mycake\cakephp3\webroot\index.php(37): Cake\Http\Server->run()
#13 {main}


2018-06-26 08:57:17 Error: [Cake\Controller\Exception\MissingActionException] Action Controller::action1() could not be found, or is not accessible.
Exception Attributes: array (
  'controller' => 'Controller',
  'action' => 'action1',
  'prefix' => '',
  'plugin' => NULL,
)
Request URL: /redirectcontroller
Stack Trace:
#0 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Http\ActionDispatcher.php(122): Cake\Controller\Controller->invokeAction()
#1 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Http\ActionDispatcher.php(96): Cake\Http\ActionDispatcher->_invoke(Object(Cake\Controller\Controller))
#2 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Http\BaseApplication.php(83): Cake\Http\ActionDispatcher->dispatch(Object(Cake\Network\Request), Object(Cake\Network\Response))
#3 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Http\Runner.php(65): Cake\Http\BaseApplication->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response), Object(Cake\Http\Runner))
#4 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Routing\Middleware\RoutingMiddleware.php(62): Cake\Http\Runner->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#5 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Http\Runner.php(65): Cake\Routing\Middleware\RoutingMiddleware->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response), Object(Cake\Http\Runner))
#6 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Routing\Middleware\AssetMiddleware.php(88): Cake\Http\Runner->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#7 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Http\Runner.php(65): Cake\Routing\Middleware\AssetMiddleware->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response), Object(Cake\Http\Runner))
#8 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Error\Middleware\ErrorHandlerMiddleware.php(81): Cake\Http\Runner->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#9 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Http\Runner.php(65): Cake\Error\Middleware\ErrorHandlerMiddleware->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response), Object(Cake\Http\Runner))
#10 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Http\Runner.php(51): Cake\Http\Runner->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#11 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Http\Server.php(92): Cake\Http\Runner->run(Object(Cake\Http\MiddlewareQueue), Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#12 C:\xampp\htdocs\mycake\cakephp3\webroot\index.php(37): Cake\Http\Server->run()
#13 {main}


2018-06-26 09:10:38 Error: [Cake\Routing\Exception\MissingControllerException] Controller class Products could not be found.
Exception Attributes: array (
  'class' => 'Products',
  'plugin' => false,
  'prefix' => false,
  '_ext' => false,
)
Request URL: /template
Stack Trace:
#0 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Http\ControllerFactory.php(72): Cake\Http\ControllerFactory->missingController(Object(Cake\Network\Request))
#1 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Http\ActionDispatcher.php(93): Cake\Http\ControllerFactory->create(Object(Cake\Network\Request), Object(Cake\Network\Response))
#2 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Http\BaseApplication.php(83): Cake\Http\ActionDispatcher->dispatch(Object(Cake\Network\Request), Object(Cake\Network\Response))
#3 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Http\Runner.php(65): Cake\Http\BaseApplication->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response), Object(Cake\Http\Runner))
#4 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Routing\Middleware\RoutingMiddleware.php(62): Cake\Http\Runner->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#5 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Http\Runner.php(65): Cake\Routing\Middleware\RoutingMiddleware->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response), Object(Cake\Http\Runner))
#6 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Routing\Middleware\AssetMiddleware.php(88): Cake\Http\Runner->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#7 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Http\Runner.php(65): Cake\Routing\Middleware\AssetMiddleware->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response), Object(Cake\Http\Runner))
#8 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Error\Middleware\ErrorHandlerMiddleware.php(81): Cake\Http\Runner->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#9 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Http\Runner.php(65): Cake\Error\Middleware\ErrorHandlerMiddleware->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response), Object(Cake\Http\Runner))
#10 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Http\Runner.php(51): Cake\Http\Runner->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#11 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Http\Server.php(92): Cake\Http\Runner->run(Object(Cake\Http\MiddlewareQueue), Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#12 C:\xampp\htdocs\mycake\cakephp3\webroot\index.php(37): Cake\Http\Server->run()
#13 {main}


2018-06-26 09:11:38 Error: [Cake\Routing\Exception\MissingControllerException] Controller class Products could not be found.
Exception Attributes: array (
  'class' => 'Products',
  'plugin' => false,
  'prefix' => false,
  '_ext' => false,
)
Request URL: /template
Stack Trace:
#0 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Http\ControllerFactory.php(72): Cake\Http\ControllerFactory->missingController(Object(Cake\Network\Request))
#1 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Http\ActionDispatcher.php(93): Cake\Http\ControllerFactory->create(Object(Cake\Network\Request), Object(Cake\Network\Response))
#2 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Http\BaseApplication.php(83): Cake\Http\ActionDispatcher->dispatch(Object(Cake\Network\Request), Object(Cake\Network\Response))
#3 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Http\Runner.php(65): Cake\Http\BaseApplication->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response), Object(Cake\Http\Runner))
#4 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Routing\Middleware\RoutingMiddleware.php(62): Cake\Http\Runner->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#5 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Http\Runner.php(65): Cake\Routing\Middleware\RoutingMiddleware->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response), Object(Cake\Http\Runner))
#6 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Routing\Middleware\AssetMiddleware.php(88): Cake\Http\Runner->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#7 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Http\Runner.php(65): Cake\Routing\Middleware\AssetMiddleware->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response), Object(Cake\Http\Runner))
#8 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Error\Middleware\ErrorHandlerMiddleware.php(81): Cake\Http\Runner->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#9 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Http\Runner.php(65): Cake\Error\Middleware\ErrorHandlerMiddleware->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response), Object(Cake\Http\Runner))
#10 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Http\Runner.php(51): Cake\Http\Runner->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#11 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Http\Server.php(92): Cake\Http\Runner->run(Object(Cake\Http\MiddlewareQueue), Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#12 C:\xampp\htdocs\mycake\cakephp3\webroot\index.php(37): Cake\Http\Server->run()
#13 {main}


2018-06-26 09:32:43 Error: Fatal Error (1): Function name must be a string in [C:\xampp\htdocs\mycake\cakephp3\src\Template\Extends\index.ctp, line 2]
Request URL: /mycake/cakephp3/extend
Trace:
Cake\Error\BaseErrorHandler::handleFatalError() - CORE\src\Error\BaseErrorHandler.php, line 216
Cake\Error\BaseErrorHandler::Cake\Error\{closure}() - CORE\src\Error\BaseErrorHandler.php, line 98
[main] - [internal], line ??



2018-06-26 09:32:43 Error: [Cake\Error\FatalErrorException] Function name must be a string
Request URL: /mycake/cakephp3/extend
Stack Trace:
#0 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Error\BaseErrorHandler.php(98): Cake\Error\BaseErrorHandler->handleFatalError(1, 'Function name m...', 'C:\\xampp\\htdocs...', 2)
#1 [internal function]: Cake\Error\BaseErrorHandler->Cake\Error\{closure}()
#2 {main}


2018-06-26 09:34:38 Error: Fatal Error (1): Function name must be a string in [C:\xampp\htdocs\mycake\cakephp3\src\Template\Extends\index.ctp, line 2]
Request URL: /mycake/cakephp3/extend
Trace:
Cake\Error\BaseErrorHandler::handleFatalError() - CORE\src\Error\BaseErrorHandler.php, line 216
Cake\Error\BaseErrorHandler::Cake\Error\{closure}() - CORE\src\Error\BaseErrorHandler.php, line 98
[main] - [internal], line ??



2018-06-26 09:34:38 Error: [Cake\Error\FatalErrorException] Function name must be a string
Request URL: /mycake/cakephp3/extend
Stack Trace:
#0 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Error\BaseErrorHandler.php(98): Cake\Error\BaseErrorHandler->handleFatalError(1, 'Function name m...', 'C:\\xampp\\htdocs...', 2)
#1 [internal function]: Cake\Error\BaseErrorHandler->Cake\Error\{closure}()
#2 {main}


2018-06-26 09:36:17 Error: Fatal Error (4): syntax error, unexpected '?>' in [C:\xampp\htdocs\mycake\cakephp3\src\Template\Extends\index.ctp, line 2]
Request URL: /mycake/cakephp3/extend
Trace:
Cake\Error\BaseErrorHandler::handleFatalError() - CORE\src\Error\BaseErrorHandler.php, line 216
Cake\Error\BaseErrorHandler::Cake\Error\{closure}() - CORE\src\Error\BaseErrorHandler.php, line 98
[main] - [internal], line ??



2018-06-26 09:36:17 Error: [Cake\Error\FatalErrorException] syntax error, unexpected '?>'
Request URL: /mycake/cakephp3/extend
Stack Trace:
#0 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Error\BaseErrorHandler.php(98): Cake\Error\BaseErrorHandler->handleFatalError(4, 'syntax error, u...', 'C:\\xampp\\htdocs...', 2)
#1 [internal function]: Cake\Error\BaseErrorHandler->Cake\Error\{closure}()
#2 {main}


2018-06-26 09:58:08 Error: [Cake\Routing\Exception\MissingControllerException] Controller class Add could not be found.
Exception Attributes: array (
  'class' => 'Add',
  'plugin' => false,
  'prefix' => false,
  '_ext' => false,
)
Request URL: /add
Referer URL: http://localhost/mycake/cakephp3/users
Stack Trace:
#0 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Http\ControllerFactory.php(72): Cake\Http\ControllerFactory->missingController(Object(Cake\Network\Request))
#1 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Http\ActionDispatcher.php(93): Cake\Http\ControllerFactory->create(Object(Cake\Network\Request), Object(Cake\Network\Response))
#2 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Http\BaseApplication.php(83): Cake\Http\ActionDispatcher->dispatch(Object(Cake\Network\Request), Object(Cake\Network\Response))
#3 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Http\Runner.php(65): Cake\Http\BaseApplication->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response), Object(Cake\Http\Runner))
#4 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Routing\Middleware\RoutingMiddleware.php(62): Cake\Http\Runner->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#5 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Http\Runner.php(65): Cake\Routing\Middleware\RoutingMiddleware->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response), Object(Cake\Http\Runner))
#6 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Routing\Middleware\AssetMiddleware.php(88): Cake\Http\Runner->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#7 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Http\Runner.php(65): Cake\Routing\Middleware\AssetMiddleware->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response), Object(Cake\Http\Runner))
#8 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Error\Middleware\ErrorHandlerMiddleware.php(81): Cake\Http\Runner->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#9 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Http\Runner.php(65): Cake\Error\Middleware\ErrorHandlerMiddleware->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response), Object(Cake\Http\Runner))
#10 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Http\Runner.php(51): Cake\Http\Runner->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#11 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Http\Server.php(92): Cake\Http\Runner->run(Object(Cake\Http\MiddlewareQueue), Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#12 C:\xampp\htdocs\mycake\cakephp3\webroot\index.php(37): Cake\Http\Server->run()
#13 {main}


2018-06-26 09:59:54 Error: Fatal Error (1): Call to undefined method App\View\AppView::link() in [C:\xampp\htdocs\mycake\cakephp3\src\Template\Users\add.ctp, line 7]
Request URL: /mycake/cakephp3/users/add
Referer URL: http://localhost/mycake/cakephp3/users/add
Trace:
Cake\Error\BaseErrorHandler::handleFatalError() - CORE\src\Error\BaseErrorHandler.php, line 216
Cake\Error\BaseErrorHandler::Cake\Error\{closure}() - CORE\src\Error\BaseErrorHandler.php, line 98
[main] - [internal], line ??



2018-06-26 09:59:55 Error: [Cake\Error\FatalErrorException] Call to undefined method App\View\AppView::link()
Request URL: /mycake/cakephp3/users/add
Referer URL: http://localhost/mycake/cakephp3/users/add
Stack Trace:
#0 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Error\BaseErrorHandler.php(98): Cake\Error\BaseErrorHandler->handleFatalError(1, 'Call to undefin...', 'C:\\xampp\\htdocs...', 7)
#1 [internal function]: Cake\Error\BaseErrorHandler->Cake\Error\{closure}()
#2 {main}


2018-06-26 10:00:44 Error: [Cake\View\Exception\MissingHelperException] Helper class ElementHelper could not be found.
Exception Attributes: array (
  'class' => 'ElementHelper',
  'plugin' => NULL,
)
Request URL: /users/add
Referer URL: http://localhost/mycake/cakephp3/users/add
Stack Trace:
#0 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Core\ObjectRegistry.php(91): Cake\View\HelperRegistry->_throwMissingClassError('Element', NULL)
#1 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\View\HelperRegistry.php(67): Cake\Core\ObjectRegistry->load('Element')
#2 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\View\View.php(905): Cake\View\HelperRegistry->__isset('Element')
#3 C:\xampp\htdocs\mycake\cakephp3\src\Template\Users\add.ctp(7): Cake\View\View->__get('Element')
#4 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\View\View.php(1014): include('C:\\xampp\\htdocs...')
#5 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\View\View.php(973): Cake\View\View->_evaluate('C:\\xampp\\htdocs...', Array)
#6 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\View\View.php(597): Cake\View\View->_render('C:\\xampp\\htdocs...')
#7 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Controller\Controller.php(617): Cake\View\View->render(NULL, NULL)
#8 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Http\ActionDispatcher.php(128): Cake\Controller\Controller->render()
#9 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Http\ActionDispatcher.php(96): Cake\Http\ActionDispatcher->_invoke(Object(App\Controller\UsersController))
#10 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Http\BaseApplication.php(83): Cake\Http\ActionDispatcher->dispatch(Object(Cake\Network\Request), Object(Cake\Network\Response))
#11 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Http\Runner.php(65): Cake\Http\BaseApplication->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response), Object(Cake\Http\Runner))
#12 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Routing\Middleware\RoutingMiddleware.php(62): Cake\Http\Runner->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#13 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Http\Runner.php(65): Cake\Routing\Middleware\RoutingMiddleware->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response), Object(Cake\Http\Runner))
#14 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Routing\Middleware\AssetMiddleware.php(88): Cake\Http\Runner->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#15 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Http\Runner.php(65): Cake\Routing\Middleware\AssetMiddleware->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response), Object(Cake\Http\Runner))
#16 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Error\Middleware\ErrorHandlerMiddleware.php(81): Cake\Http\Runner->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#17 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Http\Runner.php(65): Cake\Error\Middleware\ErrorHandlerMiddleware->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response), Object(Cake\Http\Runner))
#18 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Http\Runner.php(51): Cake\Http\Runner->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#19 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Http\Server.php(92): Cake\Http\Runner->run(Object(Cake\Http\MiddlewareQueue), Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#20 C:\xampp\htdocs\mycake\cakephp3\webroot\index.php(37): Cake\Http\Server->run()
#21 {main}


2018-06-26 10:03:24 Error: [Cake\Controller\Exception\MissingActionException] Action UsersController::users() could not be found, or is not accessible.
Exception Attributes: array (
  'controller' => 'UsersController',
  'action' => 'users',
  'prefix' => '',
  'plugin' => NULL,
)
Request URL: /users/users
Referer URL: http://localhost/mycake/cakephp3/users/add
Stack Trace:
#0 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Http\ActionDispatcher.php(122): Cake\Controller\Controller->invokeAction()
#1 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Http\ActionDispatcher.php(96): Cake\Http\ActionDispatcher->_invoke(Object(App\Controller\UsersController))
#2 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Http\BaseApplication.php(83): Cake\Http\ActionDispatcher->dispatch(Object(Cake\Network\Request), Object(Cake\Network\Response))
#3 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Http\Runner.php(65): Cake\Http\BaseApplication->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response), Object(Cake\Http\Runner))
#4 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Routing\Middleware\RoutingMiddleware.php(62): Cake\Http\Runner->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#5 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Http\Runner.php(65): Cake\Routing\Middleware\RoutingMiddleware->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response), Object(Cake\Http\Runner))
#6 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Routing\Middleware\AssetMiddleware.php(88): Cake\Http\Runner->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#7 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Http\Runner.php(65): Cake\Routing\Middleware\AssetMiddleware->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response), Object(Cake\Http\Runner))
#8 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Error\Middleware\ErrorHandlerMiddleware.php(81): Cake\Http\Runner->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#9 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Http\Runner.php(65): Cake\Error\Middleware\ErrorHandlerMiddleware->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response), Object(Cake\Http\Runner))
#10 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Http\Runner.php(51): Cake\Http\Runner->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#11 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Http\Server.php(92): Cake\Http\Runner->run(Object(Cake\Http\MiddlewareQueue), Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#12 C:\xampp\htdocs\mycake\cakephp3\webroot\index.php(37): Cake\Http\Server->run()
#13 {main}


2018-06-26 10:04:45 Error: [Cake\Controller\Exception\MissingActionException] Action UsersController::delete() could not be found, or is not accessible.
Exception Attributes: array (
  'controller' => 'UsersController',
  'action' => 'delete',
  'prefix' => '',
  'plugin' => NULL,
)
Request URL: /users/delete/9
Referer URL: http://localhost/mycake/cakephp3/users
Stack Trace:
#0 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Http\ActionDispatcher.php(122): Cake\Controller\Controller->invokeAction()
#1 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Http\ActionDispatcher.php(96): Cake\Http\ActionDispatcher->_invoke(Object(App\Controller\UsersController))
#2 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Http\BaseApplication.php(83): Cake\Http\ActionDispatcher->dispatch(Object(Cake\Network\Request), Object(Cake\Network\Response))
#3 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Http\Runner.php(65): Cake\Http\BaseApplication->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response), Object(Cake\Http\Runner))
#4 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Routing\Middleware\RoutingMiddleware.php(62): Cake\Http\Runner->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#5 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Http\Runner.php(65): Cake\Routing\Middleware\RoutingMiddleware->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response), Object(Cake\Http\Runner))
#6 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Routing\Middleware\AssetMiddleware.php(88): Cake\Http\Runner->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#7 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Http\Runner.php(65): Cake\Routing\Middleware\AssetMiddleware->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response), Object(Cake\Http\Runner))
#8 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Error\Middleware\ErrorHandlerMiddleware.php(81): Cake\Http\Runner->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#9 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Http\Runner.php(65): Cake\Error\Middleware\ErrorHandlerMiddleware->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response), Object(Cake\Http\Runner))
#10 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Http\Runner.php(51): Cake\Http\Runner->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#11 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Http\Server.php(92): Cake\Http\Runner->run(Object(Cake\Http\MiddlewareQueue), Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#12 C:\xampp\htdocs\mycake\cakephp3\webroot\index.php(37): Cake\Http\Server->run()
#13 {main}


2018-06-26 10:14:17 Error: [Cake\Datasource\Exception\RecordNotFoundException] Record not found in table "users"
Request URL: /users/delete/users/add
Referer URL: http://localhost/mycake/cakephp3/users/delete/7
Stack Trace:
#0 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\ORM\Table.php(1225): Cake\ORM\Query->firstOrFail()
#1 C:\xampp\htdocs\mycake\cakephp3\src\Controller\UsersController.php(50): Cake\ORM\Table->get('users')
#2 [internal function]: App\Controller\UsersController->delete('users', 'add')
#3 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Controller\Controller.php(435): call_user_func_array(Array, Array)
#4 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Http\ActionDispatcher.php(122): Cake\Controller\Controller->invokeAction()
#5 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Http\ActionDispatcher.php(96): Cake\Http\ActionDispatcher->_invoke(Object(App\Controller\UsersController))
#6 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Http\BaseApplication.php(83): Cake\Http\ActionDispatcher->dispatch(Object(Cake\Network\Request), Object(Cake\Network\Response))
#7 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Http\Runner.php(65): Cake\Http\BaseApplication->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response), Object(Cake\Http\Runner))
#8 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Routing\Middleware\RoutingMiddleware.php(62): Cake\Http\Runner->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#9 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Http\Runner.php(65): Cake\Routing\Middleware\RoutingMiddleware->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response), Object(Cake\Http\Runner))
#10 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Routing\Middleware\AssetMiddleware.php(88): Cake\Http\Runner->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#11 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Http\Runner.php(65): Cake\Routing\Middleware\AssetMiddleware->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response), Object(Cake\Http\Runner))
#12 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Error\Middleware\ErrorHandlerMiddleware.php(81): Cake\Http\Runner->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#13 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Http\Runner.php(65): Cake\Error\Middleware\ErrorHandlerMiddleware->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response), Object(Cake\Http\Runner))
#14 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Http\Runner.php(51): Cake\Http\Runner->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#15 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Http\Server.php(92): Cake\Http\Runner->run(Object(Cake\Http\MiddlewareQueue), Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#16 C:\xampp\htdocs\mycake\cakephp3\webroot\index.php(37): Cake\Http\Server->run()
#17 {main}


2018-06-26 10:15:59 Error: [Cake\Datasource\Exception\RecordNotFoundException] Record not found in table "users"
Request URL: /users/delete/users/add
Referer URL: http://localhost/mycake/cakephp3/users/delete/7
Stack Trace:
#0 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\ORM\Table.php(1225): Cake\ORM\Query->firstOrFail()
#1 C:\xampp\htdocs\mycake\cakephp3\src\Controller\UsersController.php(50): Cake\ORM\Table->get('users')
#2 [internal function]: App\Controller\UsersController->delete('users', 'add')
#3 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Controller\Controller.php(435): call_user_func_array(Array, Array)
#4 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Http\ActionDispatcher.php(122): Cake\Controller\Controller->invokeAction()
#5 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Http\ActionDispatcher.php(96): Cake\Http\ActionDispatcher->_invoke(Object(App\Controller\UsersController))
#6 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Http\BaseApplication.php(83): Cake\Http\ActionDispatcher->dispatch(Object(Cake\Network\Request), Object(Cake\Network\Response))
#7 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Http\Runner.php(65): Cake\Http\BaseApplication->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response), Object(Cake\Http\Runner))
#8 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Routing\Middleware\RoutingMiddleware.php(62): Cake\Http\Runner->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#9 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Http\Runner.php(65): Cake\Routing\Middleware\RoutingMiddleware->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response), Object(Cake\Http\Runner))
#10 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Routing\Middleware\AssetMiddleware.php(88): Cake\Http\Runner->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#11 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Http\Runner.php(65): Cake\Routing\Middleware\AssetMiddleware->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response), Object(Cake\Http\Runner))
#12 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Error\Middleware\ErrorHandlerMiddleware.php(81): Cake\Http\Runner->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#13 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Http\Runner.php(65): Cake\Error\Middleware\ErrorHandlerMiddleware->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response), Object(Cake\Http\Runner))
#14 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Http\Runner.php(51): Cake\Http\Runner->__invoke(Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#15 C:\xampp\htdocs\mycake\cakephp3\vendor\cakephp\cakephp\src\Http\Server.php(92): Cake\Http\Runner->run(Object(Cake\Http\MiddlewareQueue), Object(Zend\Diactoros\ServerRequest), Object(Zend\Diactoros\Response))
#16 C:\xampp\htdocs\mycake\cakephp3\webroot\index.php(37): Cake\Http\Server->run()
#17 {main}


